docs/api-reference/carto/overview.md

```diff
@@ -24,18 +27,47 @@ npm install @deck.gl/core @deck.gl/layers @deck.gl/geo-layers @deck.gl/carto
 
 ```js
 import DeckGL from '@deck.gl/react';
-import {CartoSQLLayer, setDefaultCredentials} from '@deck.gl/carto';
+import {CartoLayer, setDefaultCredentials, API_VERSIONS, MAP_TYPES} from '@deck.gl/carto';
 
 setDefaultCredentials({
+  apiVersion: API_VERSIONS.V2,
   username: 'public',
   apiKey: 'default_public'
 });
 
 function App({viewState}) {
-  const layer = new CartoSQLLayer({
+  const layer = new CartoLayer({
+    type: MAP_TYPES.QUERY,
```

Comment:
This type here isn't required for v2, right?

Reply:
Yes, it's. Not required for CartoSQLLayer